File: python/nivel 2 logica/15.py
"""
Ejercicio 15
Contar cuántas veces aparece cada número en una lista
"""

numeros = [4,15,2,4,89,15,23,2,7,4]

conteo = {}

for numero in numeros:
  if numero in conteo:
    conteo[numero] += 1
  else:
    conteo[numero] = 1

print("Conteo de Numeros: ")
for numero, cantidad in conteo.items():
  print(f"{numero} aparece {cantidad} vez/veces")

File: javascript/nivel 2 logica/15.js
/*
Ejercicio 15
Contar cuántas veces aparece cada número en una lista
*/

let numeros = [4,15,2,4,89,15,2,7,4];

let conteo = {};

for (let i = 0; i < numeros.length; i++) {
  let  numero = numeros[i];

  if(conteo[numero]){
    conteo[numero]++;
  }else{
    conteo[numero] = 1;
  }
}

console.log("Conteo de numeros");
for(let numero in conteo){
  console.log(numero + " aparece " + conteo[numero] + "vez/veces");
}